resource_processor/vmss_porter/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.8.10
```

Comment:
Can we use a base image specifically tagged with `buster`, that way we know will have az cli version available as this might get its base image changes at some point. 

https://hub.docker.com/_/python?tab=tags&page=1&ordering=last_updated&name=buster

Reply:
switched to 3.8-buster